lib/push.js

```diff
@@ -129,10 +129,26 @@ module.exports = function (options) {
 
       var files = wrench.readdirSyncRecursive('.');
       var excludedPaths = ['.build', '.git', '.whdist', '.push.zip'];
-      if (!custom)
-      {
+      if (!custom) {
         excludedPaths = excludedPaths.concat('package.json', 'node_modules', 'Gruntfile.js', 'libs', 'tasks', 'options');
       }
+      if (fs.existsSync('webhookconfig.json')) {
+        try {
+          var contents = JSON.parse(fs.readFileSync('webhookconfig.json', 'utf8'));
+        } catch(err) {
+          if (err.name === 'SyntaxError') {
+            console.log('Improperly formatted or empty webhookconfig.json'.red);
```

Comment:
Could probably be more clear about what happened here for people who aren't expecting anything - "Failed to parse config file {filename} as JSON"